spec/requests/mentees_request_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+RSpec.describe "Mentees", type: :request do
+  headers = nil
+  before do
+    user = FactoryBot.create(:user)
+    post "/login", params: { "email" => user.email, "password" => user.password }
+    user_info = JSON.parse(response.body)
+    headers = { "Authorization" => "Bearer " + user_info["token"] }
```

Comment:
This seems like a useful method for logging in a user, perhaps we can extract it so that other test suites can use it as well?

See [docs](https://relishapp.com/rspec/rspec-core/docs/helper-methods/define-helper-methods-in-a-module) for how to define a helper function.